package.json

```diff
@@ -34,11 +41,12 @@
   "license": "MIT",
   "scripts": {
     "clean": "rimraf dist",
-    "build": "node build.mjs && npm run build:types",
+    "build": "npm run clean && npx parcel build --no-source-maps && tsc --emitDeclarationOnly",
```

Comment:
Why are we using `npx` here, I think this should take up the local version of parcel even without it.